src/index.css

```diff
@@ -25,3 +33,194 @@ code {
     'Courier New',
     monospace;
 }
+
+h1 {
+  margin: 0;
+  font-size: 30px;
+}
+
+h2 {
+  width: 174px;
+  height: 24px;
+  margin: 29px 524px 19px 0;
+  font-size: 20px;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
```

Comment:
Your code appears to be WET-(Write Every Time), Microverse encourage DRY-(Don't Repeat Yourself) code, In your case; you repeated `font-stretch: normal;  font-style: normal;  line-height: normal;` a couple of times.

![image](https://user-images.githubusercontent.com/31897434/122743913-66a98f80-d27f-11eb-8f90-ac62cee28833.png)
Let's use `Combine Elements` here instead of repeating the code.

![image](https://user-images.githubusercontent.com/31897434/122744189-acfeee80-d27f-11eb-9ea7-f6d273f9646b.png)
Check out this article 👉  [good-bad-css-practices/](https://speckyboy.com/good-bad-css-practices/)